indra/newview/pipeline.cpp

```diff
@@ -117,6 +117,17 @@
 #include "llenvironment.h"
 #include "llsettingsvo.h"
 
+#ifndef LL_WINDOWS
+#define A_GCC 1
+#pragma GCC diagnostic ignored "-Wunused-function"
+#pragma GCC diagnostic ignored "-Wunused-variable"
+#if LL_LINUX
+#pragma GCC diagnostic ignored "-Wrestrict"
+#endif
+#endif
+#define A_CPU 1
+#include "app_settings/shaders/class1/deferred/CASF.glsl" // This is also C++
+
```

Comment:
Fine for now, but we should probably break this into its own .cpp file so it doesn't pollute all of pipeline.cpp.

Reply:
Fixed!

Reply:
Yeah I agree on that. Went with the easiest initial integration.